logger/src/lib.rs

```diff
@@ -75,3 +79,68 @@ pub fn setup_file_with_default(logfile: &str, filter: &str) {
         .build();
     replace_logger(logger);
 }
+
+#[cfg(unix)]
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
nit: these can get condensed into one
```suggestion
#[cfg(all(unix, not(target_arch = "wasm32")))]
```